var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -140,49 +147,44 @@ def write_makefile_inc(self):
         # TODO: test this part, it needs a full blas, scalapack and
         # partitionning environment with 64bit integers
 
-        using_xl = self.compiler.name in ['xl', 'xl_r']
         if '+int64' in self.spec:
-            if self.compiler.name == "xl" or self.compiler.name == "xl_r":
-                makefile_conf.extend(
-                    ['OPTF    = -O3',
-                     'OPTL    = %s -O3' % fpic,
-                     'OPTC    = %s -O3-DINTSIZE64' % fpic])
+            if using_xlf:
+                makefile_conf.append('OPTF = -O3')
             else:
-                makefile_conf.extend(
-                    # the fortran compilation flags most probably are
-                    # working only for intel and gnu compilers this is
-                    # perhaps something the compiler should provide
-                    ['OPTF    = %s -O  -DALLOW_NON_INIT %s' % (fpic, '-fdefault-integer-8' if self.compiler.name == "gcc" else '-i8'),  # noqa
-                     'OPTL    = %s -O ' % fpic,
```

Comment:
So you changed from -O to -O3 here?

Reply:
Yes.  As we have been using -O3 for the XL compiler suite, it felt appropriate to use -O3 for llvm Fortran when its being compiled by xlf.

Reply:
But I meant OPTL and OPTC that are now outside of the if-xl-else branch.

Reply:
@davydden Thats an oversight.  I'll push an update.

Reply:
is this fixed?

Reply:
Yes, what @davydden said